htdocs/core/modules/modSociete.class.php

```diff
@@ -197,6 +197,13 @@ public function __construct($db)
 		$this->rights[$r][3] = 0; // La permission est-elle une permission par defaut
 		$this->rights[$r][4] = 'export';
 
+		$r++;
+		$this->rights[$r][0] = 130;
+		$this->rights[$r][1] = 'Modify thirdparty information payment';
+		$this->rights[$r][3] = 0;
+		$this->rights[$r][4] = 'addupdate_thirdparty_payment_information';      // Visible if option MAIN_USE_ADVANCED_PERMS is on
```

Comment:
The key must end with   "_advance" when we add and advanced permission.
I suggest   "thirdparty_paymentinformation_advance"    (the addupdate is useless, is is already into the "write" of the next field).